docs/reviewing-release-prs.md

```diff
@@ -0,0 +1,176 @@
+
+- Changes to interfaces
+- Changes to behavior
+- Changes to runtime dependencies
+- Changes to consumer-facing documentation
+
+Conversely, you generally should not need to worry about:
+
+- Changes to internal tooling
+- Changes to developer-only dependencies
+- Changes to contributor-only documentation
+
+With that in mind, there are four categories of changes that determine the level of impact.
+
+### Breaking changes
+
+If a consumer upgrades a package to a new version without making any modifications to their code, that version is "breaking" if it causes:
+
+- An error at runtime
+- An error at compile time (e.g., a TypeScript type error)
+- An error at install time (e.g., the consumer's package manager reports a Node version incompatibility)
+- A surprising difference in behavior
+
+Given this, there are many ways a change could be regarded as breaking:
+
+- Changing a function or method to throw an error
+- Adding a required argument to a function or method
+- Adding a property to a TypeScript type
+- Narrowing the type of a property in a TypeScript type
+- Narrowing the type of an argument in a function
+- Adding a parameter to a TypeScript type
+- Removing a parameter from a TypeScript type
+- Upgrading a runtime dependency to a version which causes any of the above
+- Removing an export from the package
+- Renaming the package
+- Bumping the Node version
+- Bumping the NPM version
+
+### Additions
+
+Minimally, this category could cover any actions that extend the surface area of a package in some way:
+
+- Adding an optional property to an existing type
+- Adding an optional argument to an existing function or method
+- Adding a new export to a package
+
+However, it is important to remember that consumers care about capabilities first and interfaces second. So, some additions could also be more simply regarded as changes. For instance:
+
+- Adding a new dependency
+
+### Removals
+
+In most cases, removing something from a package will cause a breaking change, and thus should be categorized as such. There are some rare instances, however, when this is not the case:
+
+- Removing a runtime dependency
+
+### Fixes
+
+There are a few different kinds of changes that could be filed under this category.
+
+- Fixing a security vulnerability.
+- Correcting a mistake introduced in a previous release. For example, a function was supposed to be included but was not.
+- Realigning confusing or illogical behavior. This is tricky, because the longer buggy behavior is allowed to persist, the more consumers may rely upon it. In this case, it would be more thoughtful to call out a change in this behavior as breaking, even if the new behavior makes more sense. (As an example, read about the [history of MetaMask's signing methods](https://github.com/MetaMask/metamask-docs/blob/72e1a066ea1a81a7964dbc9aa83381493a2cd3eb/wallet/concepts/signing-methods.md).) But for minor cases, a "fix" may be more appropriate.
```

Comment:
The phrasing here seems to imply that "Breaking" and "Fixes" are mutually incompatible, but that's not really the case. Maybe this note about the difficulty in determining whether something is "breaking" doesn't belong in this section?

Reply:
Oh sorry, they are incompatible in this context. I was thinking of the changelog entry categories. Nevermind this comment.